sci-libs/netcdf/netcdf-4.9.2-r1.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Scientific library and interface for array oriented data access"
+HOMEPAGE="https://www.unidata.ucar.edu/software/netcdf/"
+SRC_URI="https://downloads.unidata.ucar.edu/netcdf-c/${PV}/${PN}-c-${PV}.tar.gz"
+S="${WORKDIR}"/${PN}-c-${PV}
+
+LICENSE="UCAR-Unidata"
+# SONAME of libnetcdf.so
+SLOT="0/19"
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 -riscv ~x86 ~amd64-linux ~x86-linux"
+IUSE="blosc bzip2 +dap doc examples hdf +hdf5 mpi szip test zstd"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-libs/libxml2:=
+	sys-libs/zlib:=
+	blosc? ( dev-libs/c-blosc:= )
+	bzip2? ( app-arch/bzip2:= )
+	dap? ( net-misc/curl:= )
+	hdf? (
+		media-libs/libjpeg-turbo:=
+		sci-libs/hdf:=
```

Comment:
It's odd that both hdf and hdf5 are depended on in the hdf? block.

Reply:
I had that comment in the metadata.xml changes. Duplicated it above RDEPEND.

Reply:
Do I understand correctly that `-DENABLE_NETCDF_4` enables support for hdf (the data format), and `-DENABLE_HDF4` enables support for version hdf4 (version 4 of the data format)?

I feel like it would be less confusing then to have the `hdf5` and `hdf4` use flags instead of `hdf5` and `hdf`. Should there also be `REQUIRED_USE="hdf? ( hdf5 )"`?

Reply:
1) Yes.
2) I've been told to avoid use flag renames. So that's up to someone else to decide.
3) No.